README.md

```diff
@@ -46,28 +50,27 @@ parameters of the run (
 
 # Prerequisite & Dependencies
 
-- **Summary**: For this project, we use **Docker** and **GitHub** repo to set up the dependencies and environment.
+- **Summary**: For this project, we use **Docker** and **Git** to set up the dependencies and environment.
 - **Docker image**: HumanCompatibleAI/go_attack:latest
 - **GitHub repo**:
     - Project code: https://github.com/HumanCompatibleAI/go_attack
     - Agent code: https://github.com/HumanCompatibleAI/KataGo-custom
-    - Controller code: https://github.com/Remi-Coulom/gogui/tree/v1.5.1
+    - Controller code: https://github.com/HumanCompatibleAI/gogui
 - **Setting up**
     - Download the repo
         - `git clone --recurse-submodules https://github.com/HumanCompatibleAI/go_attack.git`
-        - `cd go_attack && mkdir games`
-    - Build C++ and Python docker containers, and run the C++ container. A KataGo executable can be found in the `/engines/KataGo-custom/cpp` directory inside the container.
+        - `cd go_attack`
+    - Build C++ and Python Docker containers, and run the C++ container. A KataGo executable can be found in the `/engines/KataGo-custom/cpp` directory inside the container.
         - `docker build . -f compose/cpp/Dockerfile -t humancompatibleai/goattack:cpp`
         - `docker build . -f compose/python/Dockerfile -t humancompatibleai/goattack:python`
         - `docker run --gpus all -v ~/go_attack:/go_attack -it humancompatibleai/goattack:cpp`
     - Setup the controller
         - `cd /go_attack/controllers/gogui`
-        - `git checkout tags/v1.5.1`
-        - `./ubuntu_setup.sh`
+        - `apt update && ./ubuntu_setup.sh`
```

Comment:
what do we use `gogui` for? I've never used it besides cloning `gogui` for `twogtp` in one of the Dockerfiles for transfer experiments

Reply:
We have used it for playing games against KataGo, but it's not really a key part, and of course you can use other visualizers Sobaki instead. So we could just remove this.